backstop/tests/menu.html

```diff
@@ -429,6 +429,144 @@ <h2>Default</h2>
 
     <hr />
 
+    <h2>Skeleton</h2>
+    <section id="demo-skeleton">
+
+      <ul
+        class="iui-menu"
+        role="menu"
+      >
+        <li
+          class="iui-menu-item iui-menu-item-skeleton"
+          role="presentation"
+          style="--iui-menu-item-content-skeleton-max-width: 30%;"
```

Comment:
In #587 it uses the width of the text.  Rather than set `max-width` you could use "Lorem ipsum" or something.  Not sure if that would be considered better or worse, just thought I'd share another idea.

Reply:
But to make different width content I would need to generate some random text with selected width